### PR TITLE
Fix negative refresh timeout

### PR DESCRIPTION
**Pull request by leonid-shevtsov**
My previous changeset did not remove the need for this fix.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ export default class Transport {
 
   _scheduleRefresh(expires) {
     const now = Date.now();
-    const timeLeft = expires - now;
+    const timeLeft = expires >= now ? expires - now : 0;
```
Comment (landonwilkins) [inline src/index.js]: `Math.max(expires - now, 0)` might be slightly more clear

Comment (leonid-shevtsov) [reply src/index.js]: Feel free to release another version and bump deps in both projects. :)

Comment (landonwilkins) [reply src/index.js]: :D fair!

Comment (brightbytes-dude) [reply src/index.js]: LOL

**Pull request merged**